packages/expo-updates/ios/EXUpdates/UpdatesModule.swift

```diff
@@ -196,6 +196,17 @@ public final class UpdatesModule: Module {
         }
       }
     }
+
+    // Getter used internally by @expo/use-updates useUpdates()
+    // to initialize its state
+    AsyncFunction("nativeStateMachineContext") { (promise: Promise) in
```

Comment:
```suggestion
    AsyncFunction("getNativeStateMachineContextAsync") { (promise: Promise) in
```

(to match existing method naming pattern in this class)

Reply:
Done